src/batched-json.js

```diff
@@ -37,8 +37,8 @@ export const UrlReplacementPolicy = {
  * @param {!Element} element
  * @param {string=} opt_expr Dot-syntax reference to subdata of JSON result
  *     to return. If not specified, entire JSON result is returned.
```

Comment:
Whoa, I don't remember this file at all. I think really bad naming, `SERVICEfor` generally just gets the installed service, but this adds in a bunch of element specific code and URL replacements just for fun.

Reply:
Yea. :) Originally this code was duplicated in different extensions (anything that fetches JSON with a `src` attribute) and then extracted here. I was thinking about merging it with `batched-xhr-impl` eventually. We can let the callers provide the UrlReplacements service instead.